easybuild/easyconfigs/s/Spark/Spark-2.4.0-intel-2018b-Hadoop-2.7-Java-1.8-Python-3.6.6.eb

```diff
@@ -31,9 +31,13 @@ exts_default_options = {
 }
 
 exts_list = [
+    ('wheel', '0.33.6', {
+        'source_urls': ['https://pypi.python.org/packages/source/w/wheel'],
```

Comment:
Add 'source_urls': [PYPI_SOURCE] to exts_default_options and drop source_urls?

Reply:
@boegel Any update on this to address Ake's comments?

Reply:
fixed in f29f7abc6e